tests/test_samples.rs

```diff
@@ -208,7 +208,7 @@ fn all_the_fields() {
         }
     );
     assert_eq!(meta.workspace_members.len(), 1);
-    assert!(meta.workspace_members[0].to_string().starts_with("all"));
+    assert!(meta.workspace_members[0].to_string().contains("all"));
```

Comment:
What changed here? Can we get the original display impl back with a manual impl?

Reply:
In `stable`, that string is: `all 0.1.0 (path+file:///Users/michaelciraci/Documents/cargo_metadata/tests/all)`
In `nightly`, that string is: `path+file:///Users/michaelciraci/Documents/cargo_metadata/tests/all#0.1.0`

That's why the nightly tests were failing.